pkg/cluster/template/install/local_install.sh.go

```diff
@@ -81,16 +81,17 @@ fi
 
 chmod 755 "$bin_dir/tiup"
 
-# telemetry is not needed for offline installations
-"$bin_dir/tiup" telemetry disable
```

Comment:
Should keep

Reply:
I synced it from `install.sh`, since there is no such things in `install.sh`

Reply:
Checked again, telemetry is disabled by default. So we don't need to write it here.